.drone.yml

```diff
@@ -39,8 +39,6 @@ steps:
   image: 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci-mpn-4.1:cmdstanr
   commands:
   - R -s -e 'devtools::install_deps(upgrade = "never")'
-  - git clone -b 1.7.0 --depth 1 --single-branch https://github.com/metrumresearchgroup/bbr.git /tmp/bbr
```

Comment:
Is this now just included in the Docker container that we build/refresh with each MPN release?

Reply:
Yep, exactly.